Modify collation rules which could cause errors in regular expressions
with character ranges, as was reported in #362514 for another locale.

---
 localedata/locales/csb_PL |   15 +++++++++++----
 1 file changed, 11 insertions(+), 4 deletions(-)

--- a/localedata/locales/csb_PL
+++ b/localedata/locales/csb_PL
@@ -93,38 +93,45 @@
 
 reorder-after <U0061>
 <U0105> <aogonek>;<BAS>;<MIN>;IGNORE
-<U0104> <aogonek>;<BAS>;<CAP>;IGNORE
 <U00E3> <atilde>;<BAS>;<MIN>;IGNORE
+reorder-after <U0041>
+<U0104> <aogonek>;<BAS>;<CAP>;IGNORE
 <U00C3> <atilde>;<BAS>;<CAP>;IGNORE
 
 reorder-after <U0065>
 <U00E9> <eacute>;<BAS>;<MIN>;IGNORE
-<U00C9> <eacute>;<BAS>;<CAP>;IGNORE
 <U00EB> <ediaeresis>;<BAS>;<MIN>;IGNORE
+reorder-after <U0045>
+<U00C9> <eacute>;<BAS>;<CAP>;IGNORE
 <U00CB> <ediaeresis>;<BAS>;<CAP>;IGNORE
 
 reorder-after <U006C>
 <U0142> <lstroke>;<BAS>;<MIN>;IGNORE
+reorder-after <U004C>
 <U0141> <lstroke>;<BAS>;<CAP>;IGNORE
 
 reorder-after <U006E>
 <U0144> <nacute>;<BAS>;<MIN>;IGNORE
+reorder-after <U004E>
 <U0143> <nacute>;<BAS>;<CAP>;IGNORE
 
 reorder-after <U006F>
 <U00F2> <ograve>;<BAS>;<MIN>;IGNORE
-<U00D2> <ograve>;<BAS>;<CAP>;IGNORE
 <U00F3> <oacute>;<BAS>;<MIN>;IGNORE
-<U00D3> <oacute>;<BAS>;<CAP>;IGNORE
 <U00F4> <ocircumflex>;<BAS>;<MIN>;IGNORE
+reorder-after <U004F>
+<U00D2> <ograve>;<BAS>;<CAP>;IGNORE
+<U00D3> <oacute>;<BAS>;<CAP>;IGNORE
 <U00D4> <ocircumflex>;<BAS>;<CAP>;IGNORE
 
 reorder-after <U0075>
 <U00F9> <ugrave>;<BAS>;<MIN>;IGNORE
+reorder-after <U0055>
 <U00D9> <ugrave>;<BAS>;<CAP>;IGNORE
 
 reorder-after <U007A>
 <U017C> <zdot>;<BAS>;<MIN>;IGNORE
+reorder-after <U005A>
 <U017B> <zdot>;<BAS>;<CAP>;IGNORE
 
 reorder-end
